presentation/src/test/java/com/shifthackz/aisdv1/presentation/screen/setup/ServerSetupViewModelTest.kt

```diff
@@ -82,6 +84,12 @@ class ServerSetupViewModelTest : CoreViewModelTest<ServerSetupViewModel>() {
         } returns Single.just(mockHuggingFaceModels)
     }
 
+    @After
```

Comment:
What is the point of `unmockkAll()` here ?

Reply:
Added it with idea to possibly fix Unit-tests issue.